app/services/exporter.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+class Exporter
+  attr_accessor :metadata, :archive_path
+
+  PERSON_FIELDS_TO_EXPORT = %i[
+    name
+    city
+    phone
+    skype
+    linkedin
+    primary_tech
+    english
+    day_of_birth
+    email
+    skills
+    current_position
+    github
+    personal_email
+  ].freeze
+
+  def perform(start_time: nil, end_time: nil)
+    collect_data(start_time, end_time) && archive_data
+  end
+
+  def collect_data(start_time, end_time)
+    @metadata_fields_types = {
+      person: PERSON_FIELDS_TO_EXPORT.map { |attr| { name: attr, type: Person.columns_hash[attr.to_s].type } }
+    }
+
+    attachments = Attachment.where('name ILIKE ?', '%cv%')
```

Comment:
@rilian are we 100% sure those are all the CVs? :thinking: maybe, some dropdown with type of attachemt (or checkbox is_cv) should be used instead. Wonder whether we have any cases where cyrillic c is mixed with v

Reply:
we dont use cyrillic and most probably yes, if name contains cv it is a cv